### DISCUSSION TITLE
How to ignore className did not match for certain class?

### DISCUSSION THREAD

**Discussion by daveteu** [vercel/next.js#56573] [App Router]
Using chrome extension, I have extension that adds certain classname to body field and thus causing error to be shown. Is there anyway to ignore certain class, by settings? E.g. below, I want "expansion-alids-init" (from chrome extension) to be ignored.

![image](https://github.com/vercel/next.js/assets/18649392/3cf560fe-b3e0-4710-a6b0-2cc6acd81202)

**Comment by 2010363329** [comment]
我也有这个问题 不知道咋解决 有了解决办法能不能通知我一下

**Reply by daveteu** [reply]
没有

**Comment by icyJoseph** [comment]
If it is within your control, you can add https://legacy.reactjs.org/docs/dom-elements.html#suppresshydrationwarning to the offending DOM element
